Media Picker 3 documentation (#3106)
umbraco/UmbracoDocs: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/index.md

The markdown editor will be interpreted by the Models Builder. Behind the scenes, Umbraco uses the [Markdown NuGet package](https://www.nuget.org/packages/Markdown/).

## [Media Picker 3](Media-Picker-3/index.md)

`Alias: Umbraco.MediaPicker3`

> Media picker 3 is the latest Media Picker, which enables to select one or more medias from the Media Section. This Property Editor can be configured to only pick certain Media Types. And its possible to define Local Image Crops.

Reviewer: ```suggestion
Media Picker 3 is the latest Media Picker, which enables you to select one or more media items from the Media Section. This Property Editor can be configured to only pick certain Media Types and it's possible to define Local Image Crops.
```
